devel/doctest/Portfile

```diff
@@ -0,0 +1,18 @@
+PortSystem 1.0
+PortGroup cmake 1.0
+PortGroup github 1.0
+github.setup        onqtam doctest 2.4.6 v
```

Comment:
there should be no `v` here at the end, since upstream doesn't have this as part of the filenam

Reply:
yep, corrected, verified that we now successfully fetch (also needed move from `releases` to bare tags =\, but it works).